packages/teleport-postprocessor-js-beautify-html/src/index.ts

```diff
@@ -0,0 +1,32 @@
+// @ts-ignore
```

Comment:
let's check if @types/js-beautify exists so we can have typing support for all dependencies